### PR TITLE
[RUM][REPLAY] Migrate parent view field name to container

### PR DESCRIPTION
**Pull request by N-Boutaib**
## Motivation
Following the discussion about nested records and nested rum events naming, we are adopting `container` as a name to refer to the parent (or wrapping) view of a given nested rum browser event.

## Changes
Rename:
- `parent_view.id` to `container.view.id`
- `parent_view.source` to `container.source`

### CODE REVIEW

**Review by mquentin — APPROVED** [review]
LGTM

File: lib/cjs/generated/rum.d.ts [hunk 1]
```diff
@@ -1108,17 +1108,23 @@ export interface CommonProperties {
...
-         * ID of the parent view
+         * Attributes of the view's container
          */
-        readonly id: string;
+        readonly view: {
+            /**
+             * ID of the parent view
+             */
+            readonly id: string;
+            [k: string]: unknown;
```
Comment (ThibautGeriz) [inline lib/cjs/generated/rum.d.ts:1126]: **question:** why was this added?

File: schemas/rum/_view-container-schema.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+              "description": "ID of the parent view",
+              "pattern": "^[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}$",
+              "readOnly": true
+            }
+          },
+          "readOnly": true
+        },
+        "source": {
+          "type": "string",
+          "description": "Source of the parent view",
```
Comment (ThibautGeriz) [inline schemas/rum/_view-container-schema.json:29]: **suggestion:** rename
```suggestion
          "description": "Source of the container view",
```

**Review by ThibautGeriz — APPROVED** [review]

**Review by BenoitZugmeyer — APPROVED** [review]

**Review by amortemousque — APPROVED** [review]

**Review by hugolevyy — APPROVED** [review]
